src/models/LinksDefinition.ts

```diff
@@ -33,8 +33,8 @@ export interface LinksDefinition extends FileBase {
 
 export interface Link<T extends ConnectionOwner | ExtendedConnectionOwner> {
 	name: string;
-	from: T;
-	to: T;
+	from?: T;
+	to?: T;
 }
```

Comment:
Try to don't make theese fields optional, because it makes too much excess conditions and constructions like `link.to?.box || ''` now and in future. If you need optional fields to edit links, create new type alias with them, and use it only in editor, then remove all excess conditions you added

Reply:
It's needed not only for editor. When we get incorrect data from server the app is crash. (e.g. link: {name: 'smth' }, without `to` and `from`)